arviz/plots/kdeplot.py

```diff
@@ -146,6 +147,14 @@ def plot_kde(
 
     figsize, *_, xt_labelsize, linewidth, markersize = _scale_fig_size(figsize, textsize, 1, 1)
 
+    if isinstance(values, xr.Dataset):
+        raise ValueError(
+            "Xarray dataset object detected.Use plot_posterior,plot_denstiy,plot_joint or plot_pair"
```

Comment:
Density is spelled incorrectly, spaces between the different plots would be helpful when reading

Reply:
I'll make the changes.

Reply:
No worries, this PR is basically at the finish line. Very minor things.

Thanks so much for the contribution